packages/server-renderer/src/define-server-renderer.ts

```diff
@@ -21,8 +21,10 @@ export function defineServerRenderer(
     id: 's2:server-renderer',
 
     create: (env): SharedServerRenderer => {
-      const {rerenderWait = 50} = validateConfig(env.config) || {};
-      const serverRenderer = new ServerRenderer(serverRequest, rerenderWait);
+      const {timeout} =
+        validateConfig(env.config) || ({} as ServerRendererConfig);
```

Comment:
the or part should be handled by validateConfig

Reply:
In my opinion the validate function should not change its input. Therefore, the default should be set here.